lms/djangoapps/verify_student/migrations/0008_populate_idverificationaggregate.py

```diff
@@ -7,25 +7,7 @@
 
 
 def populate_id_verification_aggregate(apps, schema_editor):
-    ContentType = apps.get_model('contenttypes', 'ContentType')
-    IDVerificationAggregate = apps.get_model('verify_student', 'IDVerificationAggregate')
-    SoftwareSecurePhotoVerification = apps.get_model('verify_student', 'SoftwareSecurePhotoVerification')
-    SSOVerification = apps.get_model('verify_student', 'SSOVerification')
-
-    software_secure_verifications = SoftwareSecurePhotoVerification.objects.all()
-    sso_verifications = SSOVerification.objects.all()
-    for verification in chain(software_secure_verifications, sso_verifications):
-        content_type = ContentType.objects.get_for_model(verification)
-        IDVerificationAggregate.objects.create(
-            status=verification.status,
-            user=verification.user,
-            name=verification.name,
-            created_at=verification.created_at,
-            updated_at=verification.updated_at,
-            content_type=content_type,
-            object_id=verification.id,
-        )
-
+    pass
```

Comment:
Add a comment indicating why this is an empty migration now.  Maybe link to the relevant tickets?